README.md

```diff
@@ -361,20 +361,33 @@ This module integrates with Spring Security's OAuth support, all you need is the
 
 ## Proxy
 
-If you're running your application (with this okta-spring-boot dependency) from behind a network proxy, you could add JVM args to your application like:
+If you're running your application (with this okta-spring-boot dependency) from behind a network proxy, you could setup properties for it in application.yml:
+```yaml
+okta:
+  oauth2:
+    proxy:
+      host: "example-proxy.com"
```

Comment:
use `proxy.example.com` (when possible use `example.com` as this is the intended use for this domain)